automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/WorkspaceMethodConfigDetailsPage.scala

```diff
@@ -57,7 +57,7 @@ class WorkspaceMethodConfigDetailsPage(namespace: String, name: String, methodCo
     private val openLaunchAnalysisModalButtonQuery: Query = testId("open-launch-analysis-modal-button")
     private val openEditModeQuery: Query = testId("edit-method-config-button")
     private val editMethodConfigNameInputQuery: Query = testId("edit-method-config-name-input")
-    private val saveEditedMethodConfigButtonQuery: Query = testId("save-edited-method-config-button")
+    private val saveEdittedMethodConfigButtonQuery: Query = testId("save-editted-method-config-button")
```

Comment:
aaaaah the typo is killing me (but I realize we need it unless we change it everywhere)

Reply:
oh, so sorry. i will change it back to the right way: edited